lomap/tests/test_new_api.py

```diff
@@ -29,10 +29,21 @@ def basic():
 
 
 def test_generate_network_smoketest(basic):
-    network = lomap.generate_lomap_network(
-        molecules=basic,
-        mappers=lomap.LomapAtomMapper(),
-        scorer=lomap.default_lomap_score,
-    )
-
-    assert isinstance(network, gufe.LigandNetwork)
+    with pytest.deprecated_call():
+        network = lomap.generate_lomap_network(
+            molecules=basic,
+            mappers=lomap.LomapAtomMapper(),
+            scorer=lomap.default_lomap_score,
+        )
+
+        assert isinstance(network, gufe.LigandNetwork)
+
+
+def test_overdefined_deprecated_generate_network(basic):
+    with pytest.raises(ValueError):
```

Comment:
Could you add in a `match` string here please? Just making sure we picked up the right ValueError.